continuous_integration/scripts/install.sh

```diff
@@ -30,7 +30,8 @@ if [[ ${UPSTREAM_DEV} ]]; then
         git+https://github.com/dask/zict \
         git+https://github.com/dask/distributed \
         git+https://github.com/dask/fastparquet \
-        git+https://github.com/zarr-developers/zarr-python
+        git+https://github.com/zarr-developers/zarr-python \
+        git+https://github.com/graingert/PyTables@numpy-2
```

Comment:
Is this change intentional?

Reply:
Yes this has the port of pytables to numpy-2 I can swap this for a URL to the specific commit on the pytables repo instead if preferred?

Reply:
Would main of PyTables work?

Reply:
No they've not merged the fixes yet